logisland-core/logisland-api/src/main/java/com/hurence/logisland/record/Field.java

```diff
@@ -166,7 +166,26 @@ public Long asLong() {
 
     @Override
     public Boolean asBoolean() {
-        return (rawValue == null) ? null : (boolean) rawValue;
+        if(rawValue == null)
+            return null;
+
+        try{
+            boolean result = (boolean) rawValue;
```

Comment:
Cast should be checked before. Catching the exception without checking is not a good thing

Reply:
ok I'll do it